apps/login/package.json

```diff
@@ -3,15 +3,26 @@
   "private": true,
   "scripts": {
     "dev": "next dev",
-    "test": "jest",
-    "test:watch": "jest --watch",
+    "test": "concurrently --group --timings --kill-others-on-fail 'npm:test:unit' 'npm:test:integration'",
+    "test:watch": "concurrently --kill-others 'npm:test:unit:watch' 'npm:test:integration:watch'",
+    "test:unit": "jest --config ./__test__/jest.config.ts",
+    "test:unit:watch": "pnpm test:unit --watch",
+    "test:integration": "export ZITADEL_API_URL=http://localhost:22222; pnpm mock:build && concurrently --group --names 'mock,test' --success command-test --kill-others 'pnpm:mock' 'start-server-and-test start http://localhost:3000 \"test:integration:run\"'",
+    "test:integration:watch": "concurrently --names 'mock,test' --kill-others 'pnpm:mock' 'ZITADEL_API_URL=http://localhost:22222 start-server-and-test dev http://localhost:3000 \"pnpm nodemon -e js,jsx,ts,tsx,css,scss --ignore \\\"__test__/**\\\" --exec \\\"pnpm test:integration:run\\\"\"'",
+    "test:integration:run": "cypress run --config-file ./cypress/cypress.config.ts --quiet",
+    "test:integration:open": "cypress open --config-file ./cypress/cypress.config.ts",
+    "mock": "pnpm mock:build && pnpm mock:run",
+    "mock:run": "pnpm mock:destroy && docker run --rm --name zitadel-mock-grpc-server --publish 22220:22220 --publish 22222:22222 zitadel-mock-grpc-server",
+    "mock:build": "DOCKER_BUILDKIT=1 docker build --tag zitadel-mock-grpc-server ./mock",
+    "mock:build:nocache": "pnpm mock:build --no-cache",
+    "mock:destroy": "docker rm --force zitadel-mock-grpc-server 2>/dev/null",
```

Comment:
Can we describe the scripts somewhere'.. unfortunately .json does not allow comments and the contribution guide is maybe not the best place or what do you think?

Reply:
I think it's not necessary to go into details and just describe the mock commands in general in the CONTRIBUTING.md, for now?

Reply:
ok for me 👍

Reply:
done